subgraphs/v3-amm/src/utils/pricing.ts

```
@@ -1,133 +1,109 @@
-
-export const MINIMUM_ETH_LOCKED = BigDecimal.fromString('60')
-
-const Q192 = BigInt.fromI32(2).pow(192 as u8)
-export function sqrtPriceX96ToTokenPrices(sqrtPriceX96: BigInt, token0: Token, token1: Token): BigDecimal[] {
-  const num = sqrtPriceX96.times(sqrtPriceX96).toBigDecimal()
-  const denom = BigDecimal.fromString(Q192.toString())
-  const price1 = num.div(denom).times(exponentToBigDecimal(token0.decimals)).div(exponentToBigDecimal(token1.decimals))
-
-  const price0 = safeDiv(BigDecimal.fromString('1'), price1)
-  return [price0, price1]
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
```

Biome (2.1.2)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)